makefiles/arch/riscv.inc.mk

```diff
@@ -1,5 +1,19 @@
 # Target architecture for the build.
-TARGET_ARCH_RISCV ?= riscv-none-embed
+TARGET_ARCH_RISCV ?= riscv-none-elf
+
+# If TARGET_ARCH wasn't set by user, also check if riscv64-unknown-elf
+# or riscv-none-embed is present.
+ifeq (riscv-none-elf,$(TARGET_ARCH_RISCV))
+  ifeq (,$(shell which $(TARGET_ARCH_RISCV)-gcc))
+    ifneq (,$(shell which riscv64-unknown-gcc))
```

Comment:
```suggestion
    ifneq (,$(shell which riscv64-unknown-elf-gcc))
```

Reply:
With this it works out of the box for me :)

Reply:
Done and squashed in. Thanks!